pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -89,6 +89,8 @@ const (
 	cName9 = "container9-name"
 )
 
+const testPodLogDirectory = "/var/log/pods/"
```

Comment:
it will be a good idea to test with non-default path to ensure that stats are collected properly. This may create more work as test configuration scripts will need to be adjusted to mount an additional log directory correct. But without it, test coverage is not complete

Reply:
Updated tests